cookbook/specs/spec_add_side.json

```diff
@@ -0,0 +1,489 @@
+                        "stateful": 1,
+                        "hanging_api": true
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "/sideprotocol/side/interchainquery/pending_queries",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": false,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 1,
+                        "hanging_api": true
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "/sideprotocol/side/yield/params",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": false,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 1,
+                        "hanging_api": true
+                    },
+                    "extra_compute_units": 0
+                }
+                ],
+                "headers": [],
+                "inheritance_apis": [],
+                "parse_directives": [],
+                "verifications": [
+                    {
+                        "name": "chain-id",
+                        "values": [
+                            {
+                                "expected_value": "S2-testnet-1"
+                            }
+                        ]
+                    },
+                    {
+                        "name": "pruning",
+                        "values": [
+                          {
+                            "latest_distance": 14400
+                        }
+                        ]
+                    }
+                ],
+                "extensions": [
+                    {
+                        "name": "archive",
+                        "cu_multiplier": 5,
+                        "rule": {
+                            "block": 14250
+                        }
+                    }
+                ]
+            },
+              {
+                "enabled": true,
+                "collection_data": {
+                  "api_interface": "grpc",
+                  "internal_path": "",
+                  "type": "",
+                  "add_on": ""
+                },
+                "apis": [
```

Comment:
using grpcurl, the following API is missing:
grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo

Reply:
not relevant (found in COSMOSSDK)